lib/analyzer/rpm-analyzer.ts

```diff
@@ -22,18 +23,19 @@ function getPackages(targetImage: string) {
     '--qf',
     '"%{NAME}\t%|EPOCH?{%{EPOCH}:}|%{VERSION}-%{RELEASE}\t%{SIZE}\n"',
   ])
-    .catch(stderr => {
+    .catch(error => {
+      const stderr = error.stderr;
       if (typeof stderr === 'string' && stderr.indexOf('not found') >= 0) {
-        return '';
+        return { stdout: '', stderr: ''};
       }
-      throw new Error(stderr);
+      throw error;
     })
     .then(parseOutput);
 }
 
-function parseOutput(text: string) {
+function parseOutput(output: Output) {
```

Comment:
if this function will just get the `stdout` string, you won't need to import the `Output` interface here.